pom.xml

```diff
@@ -34,7 +35,7 @@
 
     <kie.download.build.version>7.48.0.Final</kie.download.build.version>
     <kogito.download.build.version>2.0.0-SNAPSHOT</kogito.download.build.version>
-    <optaplanner.download.build.version>8.4.1.Final</optaplanner.download.build.version>
+    <optaplanner.download.build.version>8.3.0.redhat-2021041404</optaplanner.download.build.version>
```

Comment:
Wouldn't it make sense here to default to an already released version? (if this one is not of course)

Reply:
That is not clear for me, could you point me out which group do you mean? 
http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/groups/

Reply:
I meant if there's an externally accessible version, but there's probably not. Disregard.

Reply:
@jstastny-cz please clarify what is externally accessible means in the context of testing product versions